staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -46,9 +96,32 @@ func RegisterMetrics() {
 	})
 }
 
-func RecordTransformation(transformationType string, start time.Time) {
+// RecordTransformation records latencies and count of TransformFromStorage and TransformToStorage operations.
+func RecordTransformation(transformationType string, start time.Time, err error) {
+	if err != nil {
+		transformerFailuresTotal.WithLabelValues(transformationType).Inc()
+		return
+	}
+
 	since := sinceInMicroseconds(start)
 	transformerLatencies.WithLabelValues(transformationType).Observe(float64(since))
+	transformerOperationsTotal.WithLabelValues(transformationType).Inc()
```

Comment:
Do you need the separate OperationsTotal metric? I'm pretty sure the histogram metric includes a total ([<basename>_count](https://prometheus.io/docs/concepts/metric_types/#histogram))

Reply:
Good point.
The plan was to use 
transformerOperationsTotal[from_strorage] and envelope_transformation_cache_misses_count to calculate cache hit rate.
I am not sure how easy it would be to reference the total from a histogram when publishing cache_hit ration internally.

Reply:
Removed OperationsTotal, will use histogram.sample_count instead.